### PR TITLE
NA-28/Add-edit-note-functionality

### PR DESCRIPTION
**Pull request by yevgeny-semak**

### CODE REVIEW

File: src/components/note-edit.component.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    const initialFormData = {
+        title: "",
+        content: "",
+    };
+
+    const [formData, setFormData] = useState(initialFormData);
+
+    useEffect(() => {
+        if (open && id !== null) {
+            apiInstance.get(`http://localhost:8000/api/notes/` + id)
```
Comment (veranika-zakharchenia) [inline src/components/note-edit.component.jsx:48]: can you move all requests to another file?

File: src/pages/notes.page.jsx [hunk 2]
```diff
@@ -31,21 +62,23 @@ const Notes = () => {
...
+        setEditNoteId(null);
+        setOpenEditModal(false);
+    };
 
     return (
         <React.Fragment>
             <SearchBar onFilter={filterNotes} />
-            <NoteAdd onAdd={addNote} />
-            <NoteList notes={filteredNotes} onDelete={deleteNote} />
+            <NoteAdd onAdd={addNote} />
```
Comment (veranika-zakharchenia) [inline src/pages/notes.page.jsx:79]: could you provide a screenshot of the homepage?

**Review by veranika-zakharchenia — CHANGES_REQUESTED** [review]
some changes are required

File: src/components/note-edit.component.jsx (hunk 1 above)
Comment (yevgeny-semak) [reply src/components/note-edit.component.jsx]: done

File: src/api.instance.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ apiInstance.interceptors.response.use(
                 console.log(tokenParts.exp);
 
                 if (tokenParts.exp > now) {
-                    return apiInstance.post(apiInstance.baseUrl + 'users/token/refresh/', {
+                    return apiInstance.post(`${apiHost}/api/users/token/refresh/`, {
```
Comment (veranika-zakharchenia) [inline src/api.instance.js:33]: for what reason did you add _api_ to the url?

Comment (yevgeny-semak) [reply src/api.instance.js:33]: since apiHost holds url for api host only
probably /api/ can be also added there

**Pull request merged**